Connectors/src/PostgreSql/Program.cs

```diff
@@ -1,12 +1,18 @@
 using PostgreSql;
 using Steeltoe.Connector.PostgreSql;
 using Steeltoe.Extensions.Configuration.CloudFoundry;
+using Steeltoe.Extensions.Configuration.Kubernetes.ServiceBinding;
 using Steeltoe.Management.Endpoint;
 
 WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
 
 // Steeltoe: Setup
+
+// Add Service Bindings when running on CloudFoundry
 builder.AddCloudFoundryConfiguration();
+
+// Add Service Bindings when running on K8S
+builder.WebHost.ConfigureAppConfiguration(builder => builder.AddKubernetesServiceBindings());
```

Comment:
I recently removed comments such as these in the connector samples. Because they only occurred on some registrations, initially I tried to describe every one of them. But I found it was hard for many of them. They often hook up multiple things. So I decided to remove those comments and place them under a "Setup" comment altogether.

I think we are better off choosing good method names, instead of having these comments. To me, "AddKubernetesServiceBindings" reads the same as "Add Service Bindings when running on K8S", so I don't think the comment is helpful. The CloudFoundry method is a little less clear, but we're planning to change that one.

Would you mind deleting these comments, or do you feel differently?

Reply:
I do like the comments ... I feel its helpful